account_asset_management/models/account_asset.py

```diff
@@ -446,7 +461,7 @@ def create(self, vals):
         asset = super().create(vals)
         if self.env.context.get("create_asset_from_move_line"):
             # Trigger compute of depreciation_base
-            asset.salvage_value = 0.0
+            asset.salvage_value = asset._get_salvage_value_profile()
```

Comment:
One doubt: is this really needed? If you remove these lines, I think it will work with the new compute.

Reply:
Thank you for your review. I have already deleted it as it was not needed.